packages/lesswrong/server/resolvers/notificationResolvers.ts

```diff
@@ -141,3 +142,21 @@ createPaginatedResolver({
     });
   },
 });
+
+addGraphQLResolvers({
```

Comment:
nit: prefer defineMutation

(this might've been written before that existed)

maybe we mark `addGraphQLResolvers` as @deprecated and suggest `defineQuery` and `defineMutation`?